app/src/main/kotlin/com/wire/android/ui/home/conversations/search/SearchUserViewModel.kt

```diff
@@ -82,7 +82,7 @@ class SearchUserViewModel @Inject constructor(
     }
 
     private suspend fun search(query: String) {
-        val (searchTerm, domain) = federatedSearchParser(query)
+        val (searchTerm, domain) = federatedSearchParser(query, addMembersSearchNavArgs?.isOtherDomainAllowed ?: true)
```

Comment:
Should really the default be true if `isOtherDomainAllowed` is null? 🤔

Reply:
that question made me to think more about the implementation and I've noticed that all the `addMembersSearchNavArgs`  might be null here (in case of creating a new conversation).

So I've updated PR and moved all the `isOtherDomainAllowed` logic directly into `SearchUserViewModel`.

Answering your question: if addMembersSearchNavArgs is null we use only default team protocol to get if other domain allowed.